[2025-06-06T01:26:40.390+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: deep_models_pipeline.cleanup_all manual__2025-06-06T01:26:39.800917+00:00 [queued]>
[2025-06-06T01:26:40.397+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: deep_models_pipeline.cleanup_all manual__2025-06-06T01:26:39.800917+00:00 [queued]>
[2025-06-06T01:26:40.398+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-06-06T01:26:40.410+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): cleanup_all> on 2025-06-06 01:26:39.800917+00:00
[2025-06-06T01:26:40.417+0000] {standard_task_runner.py:60} INFO - Started process 413 to run task
[2025-06-06T01:26:40.420+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'deep_models_pipeline', 'cleanup_all', 'manual__2025-06-06T01:26:39.800917+00:00', '--job-id', '2147', '--raw', '--subdir', 'DAGS_FOLDER/train_deep_model_dag.py', '--cfg-path', '/tmp/tmp9ifxt8l1']
[2025-06-06T01:26:40.422+0000] {standard_task_runner.py:88} INFO - Job 2147: Subtask cleanup_all
[2025-06-06T01:26:40.462+0000] {task_command.py:423} INFO - Running <TaskInstance: deep_models_pipeline.cleanup_all manual__2025-06-06T01:26:39.800917+00:00 [running]> on host b3491b3cd118
[2025-06-06T01:26:40.524+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='deep_models_pipeline' AIRFLOW_CTX_TASK_ID='cleanup_all' AIRFLOW_CTX_EXECUTION_DATE='2025-06-06T01:26:39.800917+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-06T01:26:39.800917+00:00'
[2025-06-06T01:26:40.550+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-06T01:26:40.560+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=deep_models_pipeline, task_id=cleanup_all, execution_date=20250606T012639, start_date=20250606T012640, end_date=20250606T012640
[2025-06-06T01:26:40.592+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-06T01:26:40.606+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
